pygenometracks/tracks/BigWigTrack.py

```diff
@@ -73,6 +108,14 @@ class BigWigTrack(GenomeTrack):
     def __init__(self, *args, **kwargs):
         super(self.__class__, self).__init__(*args, **kwargs)
         self.bw = pyBigWig.open(self.properties['file'])
+        self.bw2 = None
+        if 'second_file' in self.properties['operation']:
+            if self.properties['second_file'] is None:
+                raise InputError("operation: {} requires to set the parameter"
+                                 " second_file."
+                                 "".format(self.properties['operation']))
+            else:
+                self.bw2 = pyBigWig.open(self.properties['second_file'])
```

Comment:
We probably need to add a destructor like `def __del__(self):` that closes the file handle properly.

Reply:
Solved.